src/components/MainPageComponent.tsx

```diff
@@ -11,10 +27,105 @@ function MainPageComponent() {
         border: '4px solid',
         borderColor: theme.palette.primary.main,
         borderRadius: '4px',
-        height: '500px',
+        padding: '1rem',
       }}
     >
-      <Typography>Hello World</Typography>
+      {/* Grid container header of charlist, contains avatar and character name */}
+      <Grid
+        container
+        flexDirection="row"
+        justifyContent="space-between"
+        mb="1rem"
+      >
+        <Grid item xs={2}>
+          <Avatar sx={boxSx}></Avatar>
+        </Grid>
+        <Grid item xs={10}>
+          <Typography sx={{ fontSize: '5rem' }}>Hello World</Typography>
+        </Grid>
+      </Grid>
+      {/* Grid container with character stats */}
+      <Grid
+        container
+        flexDirection="row"
+        justifyContent="space-between"
+        mb="1rem"
+      >
+        {Array.from(Array(7)).map((_, index) => (
+          <Grid item xs={1.5} key={index}>
+            <Box sx={boxSx}></Box>
+          </Grid>
+        ))}
+      </Grid>
+      <Grid container>
+        {/* Grid container with character skills */}
+        <Grid
+          container
+          flexDirection="row"
+          columnSpacing={3}
+          justifyContent="space-between"
+          mb="1rem"
+        >
+          <Grid item xs={4}>
+            <Grid container height="100%" flexDirection="column">
+              {Array.from(Array(17)).map((_, index) => (
+                <Grid item key={index}>
+                  <Grid
+                    container
+                    flexDirection="row"
+                    justifyContent="space-between"
+                  >
+                    {Array.from(Array(3)).map((_, index) => (
+                      <Box sx={{ ...boxSx }} key={index}></Box>
```

Comment:
`key={index}` is bad. I see, you can't do anything about it here, but maybe consider using `uuidv4` lib?

Reply:
Ok, but I think in future we dispose of components in array.